components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -43,7 +43,7 @@
           Name changes are not allowed for the default project.
         </span>
         <div id="button-bar" *ngIf="!isChefManaged">
-          <chef-button [disabled]="isLoading || !projectForm.valid || projectForm.controls['name'].value === project?.name"
+          <chef-button [disabled]="isLoading || !projectForm.valid || !projectForm.dirty"
```

Comment:
... it obviates the need for any custom code--just always check the form `dirty` bit.